samples/nrf9160/download/sample.yaml

```diff
@@ -0,0 +1,8 @@
+sample:
+  name: Download sample
+tests:
+  samples.nrf9160.download_client:
```

Comment:
I think we should add another build variant that would enable CoAP, otherwise it wouldn't be built by the CI at this point. I also wonder about the shell module, it doesn't seem to be built by anything.

Reply:
Good point, I added the `overlay-ci.conf` with `SHELL`, DOWNLOAD_CLIENT_SHELL` and `COAP`.

Reply:
You can specify extra configs directly in the `sample.yaml` file, I learned that not long ago:
https://github.com/nrfconnect/sdk-nrf/blob/master/applications/serial_lte_modem/sample.yaml#L14
IMO a cleaner solution than adding an extra overlay file.

Reply:
Nice, shall I have only one entry in `tests` that builds everything instead of having two?

Reply:
I think one entry should be enough, sanitycheck does not run any tests on the sample, we just want to make sure it builds.